src/include/symbol_table/symbole_table.h

```diff
@@ -0,0 +1,101 @@
+#ifndef JUCC_SYMBOL_TABLE_SYMBOL_TABLE_H
+#define JUCC_SYMBOL_TABLE_SYMBOL_TABLE_H
+
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+namespace jucc {
+
+struct Node {
+  /**
+   * Used to store the name of the identifier
```

Comment:
use C++ class

Reply:
Why a struct seems good, why is a class needed ?

Reply:
uniformity, dear :)